shap/benchmark/perturbation.py

```diff
@@ -93,15 +103,14 @@ def score(self, attributions, X, y=None, label=None, silent=False):
             
         self.score_values.append(np.array(svals))
         
-        if self.sort_order == "negative":
+        if self.sort_order == "negative": 
```

Comment:
I think strings like "negative" should be moved to a constant

Reply:
Yep, moving things into string file is another ongoing process and will be taken care of in the next PR :)